docs/swagger.yaml

```diff
@@ -165,7 +165,7 @@ paths:
           required: true
           schema:
             type: string
-            default: '{{reference}}'
+            default: '{{orderId}}'
```

Comment:
are'nt we calling this `reference` when talking externally?

Reply:
Hm, maybe we do, but it's called orderId in the global postman environment, and the postman collection tool uses the default parameter from the swagger, so I think it's best avoid having to remember this all the time